ui/panels-plugin/src/plugins/bar-chart/bar-chart-model.ts

```diff
@@ -16,6 +16,8 @@ import { CalculationType, Definition, UnitOptions } from '@perses-dev/core';
 import { OptionsEditorProps } from '@perses-dev/plugin-system';
 
 export const DEFAULT_UNIT: UnitOptions = { kind: 'Decimal' };
```

Comment:
maybe we should also set `abbreviate` to true similar to TimeSeriesChart

Reply:
Oh that's a good call, `abbreviate` is probably what users will want more of time (it means we'll use shortened, more readable values like `10K` instead of `10,000`)

Reply:
Good idea, added `abbreviate` [caae104](https://github.com/perses/perses/pull/1325/commits/caae1049f423d9b6bde6d35ac829883109239b7d)